specification/resources/resource-manager/Microsoft.Management/preview/2018-01-01-preview/management.json

```diff
@@ -0,0 +1,1266 @@
+    },
+    "ExpandParameter": {
+      "name": "$expand",
+      "in": "query",
+      "required": false,
+      "type": "string",
+      "enum": [
+        "children"
+      ],
+      "x-ms-parameter-location": "method",
+      "description": "The $expand=children query string parameter allows clients to request inclusion of children in the response payload."
+    },
+    "RecurseParameter": {
+      "name": "$recurse",
+      "in": "query",
+      "required": false,
+      "type": "boolean",
+      "x-ms-parameter-location": "method",
+      "description": "The $recurse=true query string parameter allows clients to request inclusion of entire hierarchy in the response payload. Note that  $expand=children must be passed up if $recurse is set to true."
+    },
+    "CreateGroupRequestParameter": {
+      "name": "createGroupRequest",
+      "in": "body",
+      "required": true,
+      "description": "Management group creation parameters.",
+      "schema": {
+        "$ref": "#/definitions/CreateManagementGroupRequest"
+      }
+    },
+    "PatchGroupRequestParameter": {
+      "name": "createGroupRequest",
+      "in": "body",
+      "required": true,
+      "description": "Management group patch parameters.",
+      "schema": {
+        "$ref": "#/definitions/PatchManagementGroupRequest"
+      }
+    },
+    "FilterParameter": {
+      "name": "$filter",
+      "in": "query",
+      "required": false,
+      "description": "A filter which allows the exclusion of subscriptions from results (i.e. '$filter=children.childType ne Subscription')",
+      "type": "string",
+      "x-ms-parameter-location": "method"
+    },
+    "GroupNameParameter": {
+      "name": "groupName",
+      "in": "query",
+      "required": false,
+      "description": "A filter which allows the call to be filtered for a specific group.",
+      "type": "string",
+      "x-ms-parameter-location": "method"
+    },
+    "OperationResultIdParameter": {
+      "name": "operationResultId",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "enum": [
+        "create",
+        "delete"
+      ],
+      "description": "The id of the operation result."
+    },
+    "CheckNameAvailabilityParameter": {
+      "name": "checkNameAvailabilityRequest",
+      "in": "body",
+      "required": true,
+      "description": "Management group name availability check parameters.",
+      "schema": {
+        "$ref": "#/definitions/CheckNameAvailabilityRequest"
+      }
+    },
+    "SkipTokenParameter": {
+      "name": "$skiptoken",
+      "in": "query",
+      "required": false,
+      "type": "string",
+      "description": "Page continuation token is only used if a previous operation returned a partial result. \nIf a previous response contains a nextLink element, the value of the nextLink element will include a token parameter that specifies a starting point to use for subsequent calls.\n"
+    },
+    "CacheControlHeader": {
```

Comment:
>CacheControlHeader [](start = 5, length = 18)

Is this required?

Reply:
Yes.  There's a bug with arm pas caching that this bypasses.

Reply:
Why not mark it as required then?

---
In reply to: [164897820](https://github.com/Azure/azure-rest-api-specs/pull/2345#discussion_r164897820) [](ancestors = 164897820)

Reply:
Will do.